src/ledger/LedgerTxnAccountSQL.cpp

```diff
@@ -541,6 +473,7 @@ LedgerTxnRoot::Impl::dropAccounts()
            "homedomain         VARCHAR(32)  NOT NULL,"
            "thresholds         TEXT         NOT NULL,"
            "flags              INT          NOT NULL,"
+           "signers            TEXT,"
            "lastmodified       INT          NOT NULL"
            ");";
     mDatabase.getSession() << "CREATE TABLE signers"
```

Comment:
I don't think we need this or the index creation code below.

Reply:
good catch